geoportailv3/static/js/mymapsservice.js

```diff
@@ -470,9 +470,8 @@ app.Mymaps.prototype.loadFeatures_ = function() {
 
 /**
  * update the map with layers
- * @private
  */
```

Comment:
I don't think so. 
I don't want to prevent renaming the function. I just want having the function available outside the class. (not private)